test/e2e/gcp/cluster_upgrade.go

```diff
@@ -138,7 +138,7 @@ var _ = SIGDescribe("Upgrade [Feature:Upgrade]", func() {
 			framework.ExpectNoError(err)
 
 			testSuite := &junit.TestSuite{Name: "Cluster upgrade"}
-			clusterUpgradeTest := &junit.TestCase{Name: "[sig-cluster-lifecycle] cluster-upgrade", Classname: "upgrade_tests"}
+			clusterUpgradeTest := &junit.TestCase{Name: "[sig-cloud-provider-gcp] cluster-upgrade", Classname: "upgrade_tests"}
```

Comment:
No such sig. I think its either sig-cloud-provider or the appropriate GCP WG.

Reply:
as per @justaugustus 's comment on slack:
```
It's a subproject, so if we're being super correct, it's sig-cloud-provider-provider-gcp, but that's shortened to sig-cloud-provider-gcp in test-infra
```
https://app.slack.com/team/U0E0E78AK

so in a way this current tag is valid for the sub-project.